src/tests/transformers/fixtures/scoped-source.html

```
@@ -0,0 +1 @@
+<div class="tracking-widest">Scoped source fixture</div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)